### DISCUSSION TITLE
MP4/H.264 encoding

### DISCUSSION THREAD

**Discussion by jrodh13** [automatic-ripping-machine/automatic-ripping-machine#1135] [Q&A]
Is there a way to get the final output to be in a more standard format such as MP4 with the H.264 encoding? I'd like to play the files natively on almost any device without the need for second party software.

**Comment by SylvainMT** [comment]
See my reply to this question here https://github.com/automatic-ripping-machine/automatic-ripping-machine/discussions/1134#discussioncomment-9575538
